mailpoet/lib/Doctrine/SerializableConnection.php

```diff
@@ -47,16 +47,4 @@ public function rollBack() {
       throw $e;
     }
   }
-
-  public function handleExceptionDuringQuery(Throwable $e, string $sql, array $params = [], array $types = []): void {
```

Comment:
Thanks for pointing out it! I wanted to test, but in the end, I forgot.

This solution is no longer possible because we updated Doctrine DBAL from 2 to 3, but I added a new one in [this commit](https://github.com/mailpoet/mailpoet/pull/5641/commits/76f457eb2c087732a2d5e0987da4d1db5c22f9cd).

Reply:
Thank you for adding this code!